src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -1447,7 +1447,18 @@ class DotVVM {
         ko.virtualElements.allowedBindings["withGridViewDataSet"] = true;
         ko.bindingHandlers["withGridViewDataSet"] = {
             init: makeUpdatableChildrenContextHandler(
-                (bindingContext, value) => bindingContext.extend({ $gridViewDataSet: value, [foreachCollectionSymbol]: dotvvm.evaluator.getDataSourceItems(value) }),
+                (bindingContext, value, allBindings) => bindingContext.extend({
+                    $gridViewDataSet: value,
+                    [foreachCollectionSymbol]: dotvvm.evaluator.getDataSourceItems(value),
+                    $gridViewDataSetHelper: {
+                        columnMapping: allBindings.get("gridViewDataSetColumnMapping"),
+                        isInEditMode: function ($context) {
+                            let columnName = $context.$gridViewDataSet().RowEditOptions().PrimaryKeyPropertyName();
+                            columnName = this.columnMapping[columnName] || columnName;
+                            return ko.unwrap($context.$gridViewDataSet().RowEditOptions().EditRowId) === ko.unwrap($context.$data[columnName]);
```

Comment:
Is there a case when `EditRowId` is not observable? That would clearly be a bug - we should not use `ko.unwrap` here